src/Compat.jl

```diff
@@ -1130,6 +1130,24 @@ if VERSION < v"1.8.0-DEV.1016"
     export chopprefix, chopsuffix
 end
 
+# https://github.com/JuliaLang/julia/pull/59425
+if VERSION < v"1.13.0-DEV.1078"
```

Comment:
Ah, probably need this
```suggestion
if VERSION < v"1.13.0-DEV.1078"
    if VERSION >= v"1.8.0-DEV.1016"
        # Extend from Base without qualification (avoids duplicating the definitions)
        import Base: chopprefix, chopsuffix
    end
```